src/store/slices/node/actionsAsync.ts

```diff
@@ -114,11 +114,11 @@ const getAddressesThunk = createAsyncThunk<
       native: string;
     }
   | undefined,
-  BasePayloadType,
+  { payload: BasePayloadType; force?: boolean },
```

Comment:
lets extend here BasePayloadType by force, it will be a lot easier to use the thunk later on